src/components/label/Label.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+import { useTheme } from '../../styles/themes';
+
+import { Body } from '../typography/Body';
+
+type LabelType = 'outlined' | 'filled' | 'soft';
+type LabelColor =
+    | 'error'
+    | 'warning'
+    | 'success'
+    | 'information'
+    | 'secondary'
+    | 'primary'
+    | 'neutral';
+
+interface Props {
+    style?: ViewStyle;
+    text: string;
+    type: LabelType;
+    labelColor?: LabelColor;
+}
+
+export const Label = ({ style, text, type, labelColor = 'neutral' }: Props) => {
+    const theme = useTheme();
+
+    const transparencyValue = '29';
```

Comment:
Where does that value come from ? I see a 16% in the design kit

Reply:
https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

i picked it from here, i need a 16% transparency, so it equals to 29